src/components/TextArea/TextArea.css

```diff
@@ -0,0 +1,47 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
```

Comment:
let's not duplicate this CSS file. let's get TextField merged so these components can share styles

Reply:
agreed